### DISCUSSION TITLE
Occasionally getting a health check (Network is up transmission-daemon process not running) - Options to avoid this going forward?

### DISCUSSION THREAD

**Discussion by andsoitgoes** [haugene/docker-transmission-openvpn#2320] [Q&A]
Hi all,

this is new to me over the last few months, I've recently upgraded my Pi to a 64 bit OS and I've loaded everything back to my system, I've had to roll back to Transmission 4.0 (the latest and development flavours just didn't work) but every week or so, I'll get a health check that comes back with "Network is up transmission-daemon process not running"

I restart the container and everything is fine again, as usual.

Is there a way to avoid this? I've tried to upgrade to the dev/master branches however I still get the unbound variable issue I'd reported previously, so I'm having to stick with the basic 4.0 release from September, would love to upgrade but I don't know how to get past that error.

Or this one - Is it worth it setting something up so that when Transmission fails the health check (this is always the only failure) that I could just have the container restart?

Would love some thoughts of people who might know what's going on in the background, thanks!

**Comment by edgd1er** [comment]
Hi,
Unbound error is due to an unset variable. Could you copy the logs?
As a workaround you may define empty the unbound variable.

**Reply by andsoitgoes** [reply]
Okay, so this is interesting. 

I loaded it up again, but didn't do Dev this time - just did latest.

It.... works. No errors.

Not sure what happened or what changed, not gonna complain - I'll see if I run into the health check issue again. If I do, are there any solutions for that?
